xcode/Subconscious/Shared/Components/Common/Profile/UserProfileView.swift

```diff
@@ -381,6 +394,14 @@ private struct EditProfileSheetModifier: ViewModifier {
                     )
                 }
             }
+            .onReceive(store.actions) { action in
+                switch(action) {
+                case .succeedEditProfile:
+                    app.send(.fetchNicknameFromProfile)
+                case _:
+                    break
+                }
+            }
```

Comment:
It's not _strictly_ related to the issue this PR is tackling, but I realised that the nickname could end up out-of-sync with `AppModel.nickname` while testing this fix.

I've adopted the proposed approach in #637 instead of implementing several layers of `notify` here.